build.gradle

```diff
@@ -5,6 +5,10 @@ plugins {
 	id 'org.sonarqube' version '3.1.1'
 }
 
+compileJava {
+	options.compilerArgs += ["--enable-preview"]
```

Comment:
I assume you added this option to enable `records` feature but since we are not using it anymore I think you can remove it

Reply:
> I assume you added this option to enable `records` feature but since we are not using it anymore I think you can remove it

Maybe we can use them somewhere else, so, if it's okay, can I not remove the preview features? :)